docs/releasing-files.md

```diff
@@ -287,7 +287,6 @@ Please run through this checklist before making a review request.
 4. Have you [redacted any low counts](#redacting-counts-less-than-or-equal-to-7)?
 5. Have you [rounded any counts](#rounding-counts) (including [counts underlying rates](#rounding-rates))?
 6. Have you supplied underlying counts for all of your results?
-7. Are all of the outputs clearly described?
```

Comment:
Is this deletion intentional, and if so why?

Reply:
This was accidental, i've reinstated it